src/clj/rems/db/test_data.clj

```diff
@@ -63,6 +66,7 @@
    "RDapprover1@funet.fi" {:eppn "RDapprover1@funet.fi" :mail "RDapprover1.test@rems_example.org" :commonName "RDapprover1 REMSDEMO"}
    "RDapprover2@funet.fi" {:eppn "RDapprover2@funet.fi" :mail "RDapprover2.test@rems_example.org" :commonName "RDapprover2 REMSDEMO"}
    "RDreview@funet.fi" {:eppn "RDreview@funet.fi" :mail "RDreview.test@rems_example.org" :commonName "RDreview REMSDEMO"}
+   "RDorganizationowner@funet.fi" {:eppn "RDorganizationowner@funet.fi" :mail "RDorganizationowner.test@test_example.org" :commonName "RDorganizationowner REMSDEMO" :organization "organization"}
```

Comment:
does this user exist in test haka?

Reply:
Apparently not, I amended the patch, not adding this user to this particular list.